.github/workflows/pr-build-test.yml

```
@@ -0,0 +1,161 @@
+          # Upload blueprint to transfer.sh
+          BLUEPRINT_URL=$(curl --upload-file pr-blueprint.json https://transfer.sh/pr-${{ github.event.pull_request.number }}-blueprint.json)
+          echo "blueprint_url=$BLUEPRINT_URL" >> $GITHUB_OUTPUT
+
+          # Create Playground URL with blueprint parameter
+          PLAYGROUND_URL="https://playground.wordpress.net/?blueprint-url=${BLUEPRINT_URL}"
+          echo "url=$PLAYGROUND_URL" >> $GITHUB_OUTPUT
+
+      - name: Comment on PR
+        uses: actions/github-script@v7
+        with:
+          script: |
```

actionlint (1.7.9)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)